src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,104 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private int capacity = DEFAULT_CAPACITY;
+    private int threshold = (int) (DEFAULT_CAPACITY * LOAD_FACTOR);
+    private Node<K, V>[] table;
```

Comment:
initialize this field in the constructor, this would help you avoid a few null checks